docs/UserGuide.md

```diff
@@ -11,42 +11,61 @@
 1. **git `2.14`** or later on the command line.
  > Type `git --version` on your OS terminal and ensure that you have the correct version of **git**.
 
-## How to Generate Dashboard
+## How to Generate Report
 1. Download the latest executable Jar on our [release](https://github.com/reposense/RepoSense/releases/latest).
    * Alternatively, you can compile the executable Jar yourself by following our [build from source guide](Build.md).
 1. Execute it on the OS terminal. <br>
 Usage: `java -jar RepoSense.jar -config CSV_CONFIG_FILE_PATH [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY] [-formats FORMAT...]`
 1. The report will be generated in the designated OUTPUT_DIRECTORY, or current working directory otherwise.
-1. To visualize the report, open `index.html`.
-1. If the dashboard was not loaded automatically, upload the `archive.zip` (generated in the OUTPUT_DIRECTORY) manually to load the data.
 
-Sample usage:
+Sample usage to generate the report:
 ```
 $ java -jar RepoSense.jar -config CSV_path.csv -output output_path/ -since 01/10/2017 -until 01/11/2017 -formats java adoc js
 ```
-
 Argument List:
 - config : Mandatory. The path to the CSV config file.
 - output : Optional. The path to the dashboard generated. If not provided, it will be generated in the current directory.
 - since : Optional. The start date of analysis. Format: `DD/MM/YYYY`
 - until : Optional. The end date of analysis. Format: `DD/MM/YYYY`
 - formats : Optional. The file formats to analyse. Formats: `alphanumerical file formats`. If not provided, the following file formats will be used. `adoc, cs, css, fxml, gradle, html, java, js, json, jsp, md, py, tag, xml`
 
+## How to View Dashboard
+# With jar
+1. Ensure that you have generated the report.
+1. Execute it on the OS terminal. <br>
+Usage `java -jar RepoSense.jar -view REPORT_DIRECTORY`
+
+```
+java -jar RepoSense.jar -view report_path/
```

Comment:
hmm, maybe lets continue the example from generate report,
e.g. -view output_path/

Btw, since we output the report as `reposense-report/`, do we use `output_path/` or `output_path/reposense-report/`?

Reply:
If we get report generator to print the absolute directory, perhaps we can leave it as `output_path/`.
Else we can append `reposense-report/` internally, so the user can just reuse the output directory he gave? If so, need to update the guide and instructions to provide the path to the folder containing the `reposense-report/` folder too.

Reply:
output_path/reposense-report/

Reply:
oh well, for now just update the userguide to `output_path/reposense-report/` then